src/compounds/tabs/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
why would we not have a package.lock, is this a pattern on this repo?

Reply:
I believe this is a pattern, check the `_element-template`

Reply:
Libraries don't need lockfiles - they were causing issues with the build (remember the double PRs and separate commits with package locks after publishing?) and they're not used when the package is installed anyway.